featureflags/analytics.py

```diff
@@ -23,7 +24,7 @@
 FEATURE_IDENTIFIER_ATTRIBUTE = 'featureIdentifier'
 FEATURE_NAME_ATTRIBUTE = 'featureName'
 VARIATION_IDENTIFIER_ATTRIBUTE = 'variationIdentifier'
-VARIATION_VALUE_ATTRIBUTE = 'featureValue'
+VARIATION_VALUE_ATTRIBUTE = 'variationValue'
```

Comment:
`featureValue` works and has been left in ff-server to be backwards compatible with the Java SDK which send it like this. But I thought while I'm here, make it use the new attribute. Have tested and flag metrics are still working.